code/game/machinery/doors/door.dm

```diff
@@ -30,6 +30,11 @@
 	var/repairing = 0
 	var/block_air_zones = 1 //If set, air zones cannot merge across the door even when it is opened.
 	var/close_door_at = 0 //When to automatically close the door, if possible
+	// VOREStation Edit Start
+	var/tintable = 0
+	var/icon_tinted
+	var/id_tint
+	// VOREStation Edit End

```

Comment:
This could be in door_vr
/obj/machinery/door
	var/tintable = 0
	var/icon_tinted
	var/id_tint